Makefile

```diff
@@ -62,16 +62,47 @@ draw_deps:
 ########################################
 ### Testing
```

Comment:
can we get a single makefile target to run all the tests so we can easily run the full suite locally? need to ensure its easy to see logs and debug when doing this so we can figure out issues ...

Reply:
done